tools/src/h5diff/ph5diff_main.c

```diff
@@ -241,13 +254,14 @@ void
 print_manager_output(void)
 {
     /* If there was something we buffered, let's print it now */
-    if ((outBuffOffset > 0) && g_Parallel) {
-        printf("%s", outBuff);
+    if (g_Parallel) {
+        if (outBuffOffset > 0)
+            printf("%s", outBuff);
```

Comment:
Fixes another issue of overflow file contents sometimes getting dropped when `outBuffOffset == 0`.